ethsigner/signer/multiplatform/src/main/java/tech/pegasys/ethsigner/signer/multiplatform/FileBasedSigningMetadataFile.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package tech.pegasys.ethsigner.signer.multiplatform;
+
+import java.nio.file.Path;
+
+import com.google.common.base.Objects;
+
+class FileBasedSigningMetadataFile {
```

Comment:
What's the difference between the KeyPasswordFile and this class other than the name? The properties it provides looks the same. Does it make sense to use the same datatype in both places where these classes are used?

Reply:
once the other types are added there will be an inheritance structure